packages/fx-core/templates/plugins/resource/bot/bicep/botConfiguration.template.v2.bicep

```diff
@@ -3,11 +3,9 @@
 @secure()
 param provisionParameters object
 param provisionOutputs object
-@secure()
```

Comment:
we need @secure() for currentAppSettings because it also contains secrets

Reply:
sure